.github/actions/spelling/excludes.txt

```diff
@@ -17,3 +17,6 @@ dotnet-tools.json$
 appsettings.json$
 package.json$
 package-lock.json$
+.psm1$
```

Comment:
Why is there a check-spelling and spelling folder too? We should just have one.

Reply:
Now the workflow is `check-spelling`, the action is `spelling`. Use `check-spelling` for the action too.